cavityDisplayGUI.py

```diff
@@ -85,24 +91,34 @@ def __init__(self, parent=None, args=None):
 
                     severityPV = PV(cavityObject.pvPrefix + SEVERITY_SUFFIX)
                     statusPV = PV(cavityObject.pvPrefix + STATUS_SUFFIX)
+                    descriptionPV = PV(cavityObject.pvPrefix + DESCRIPTION_SUFFIX)
 
-                    # This line is meant to initialize the cavityWidget colors and shapes when first launched
+                    # These lines are meant to initialize the cavityWidget color, shape, and descriptionPV values when first launched
                     self.severityCallback(cavityWidget, severityPV.value)
                     self.statusCallback(cavityWidget, statusPV.value)
+                    cavityWidget.setToolTip('initial fault description')
```

Comment:
Is this needed?

Reply:
Nope, not needed! I put it there before I realized that the hover text was blank simply because the PV CUDDESC hadn't changed and therefore had not been written too. I thought I had to initialize it with something lol. This line is now obsolete